src/main/java/frc/robot/RobotContainer.java

```diff
@@ -88,6 +89,8 @@
     private final Hood m_hoodSubsystem  = Hood.getInstance();
     private final Shooter m_shooterSubsystem = Shooter.getInstance();
     private final IntakeFeeder m_intakeSubsystem = IntakeFeeder.getInstance();
+
+    private final Intake m_intake = Intake.getInstance();
```

Comment:
Let's have the intake feeder actually own the motorized intake, so there's a single interface for everything.